manifests/params.pp

```diff
@@ -40,7 +40,14 @@
 
     'RedHat': {
       $dhcp_dir    = '/etc/dhcp'
-      $packagename = 'dhcp'
+      if $facts['os']['name'] == 'Fedora' {
+        $packagename = 'dhcp'
+      } else {
+        $packagename = $facts['os']['release']['major'] ? {
```

Comment:
It's considered bad practice not to have a default case. I'd also check if on modern Fedora (we should update the versions we support in this module) also made the rename. In that case `dhcp-server` can be the default case.

Reply:
This sounds like a better approach for sure. I will rework it.